rpc/backend/blocks.go

```diff
@@ -160,7 +160,8 @@ func (b *Backend) GetBlockTransactionCount(block *tmrpctypes.ResultBlock) *hexut
 	}
 
 	ethMsgs := b.EthMsgsFromTendermintBlock(block, blockRes)
-	n := hexutil.Uint(len(ethMsgs))
+	endBlockMsgs := b.EthMsgsFromTendermintEndBlock(blockRes)
```

Comment:
Because `EthMsgsFromTendermintBlock` is called in other places too which won't be able to handle our custom eth messages which are missing certain fields like sender signature etc.

Reply:
that makes sense. what's an example of that?